### PR TITLE
Add routes for projects and helpers for all models

### PR DESCRIPTION
**Pull request by carpecodeum**
# Description

Add project routes and controllers and minor changes to the usercontrollers. Add helper functions for all models.

Fixes #424 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by carpecodeum** [comment]
@ivantha @Tushar19varshney @UdeshUK please review and tell me if anything needs to be changed

**Comment by UdeshUK** [comment]
@carpecodeum can you fix the conflicts?

File: labellab-flask/api/serializers/project.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from api.extensions import db, ma
+
+class ProjectSchema(ma.ModelSchema):
+    """
+    Serializer class for project
+    """
+        
+    id = fields.Int(dump_only=True) 
+    projectname = fields.Str()
+    projectdescription = fields.Str()
```
Comment (hiranthaR) [inline labellab-flask/api/serializers/project.py:12]: This should be project_name and project_description, no?

File: labellab-flask/api/serializers/team.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from api.extensions import db, ma
+from api.serializers.projectmember import ProjectMemberSchema
+
+class TeamSchema(ma.ModelSchema):
+    """
+    Serializer class for team
+    """
+        
+    id = fields.Int(dump_only=True) 
+    teamname = fields.Str()
```
Comment (hiranthaR) [inline labellab-flask/api/serializers/team.py:12]: team_name

Comment (carpecodeum) [reply labellab-flask/api/serializers/team.py:12]: Yes these were suggested by @Tushar19varshney  in #447 I have referenced it there.

**Pull request merged**